docker/modules/Dockerfile.faux1

```diff
@@ -30,6 +30,12 @@ RUN sh ssh_build.sh
     
 FROM daqf/aardvark:latest
 
+# wpa_suplicant for dot1x
+
+RUN $AG update && $AG install wpasupplicant
```

Comment:
this too -- in the wrong spot in this file.  Look at the "FROM" directives and see how they're used, and then how docker layers are shared across builds.

Reply:
Yup. My bad. Changed.